src/BlazorUI/Bit.BlazorUI/Components/Notifications/SnackBar/BitSnackBar.scss

```
@@ -42,28 +100,53 @@
     position: absolute;
     height: spacing(0.5);
     inset-inline-start: 0;
+    animation-fill-mode: forwards;
     animation-timing-function: linear;
-    background-color: var(--bit-snb-fg);
+    background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)